reccmp/isledecomp/compare/core.py

```diff
@@ -337,7 +337,10 @@ def orig_bin_checker(addr: int) -> bool:
 
             for fun in codebase.iter_name_functions():
                 batch.set_orig(
-                    fun.offset, type=EntityType.FUNCTION, stub=fun.should_skip()
+                    fun.offset,
+                    type=EntityType.FUNCTION,
+                    stub=fun.should_skip(),
+                    library=fun.is_library(),
```

Comment:
I'd also go for `is_library_function` here

Reply:
Same as my previous comment, but with the added point that adding `is_` to the beginning might also be consistent. For example, the line above it uses `stub`, not `is_stub`. It just seems more consistent to me to use `library` or even `library_function` here to match.